sources/osgUtil/ComposerPostProcess.js

```diff
@@ -118,6 +118,10 @@ var ComposerPostProcess = function() {
     // in case of dynamic resolution scaling : true will do a final upsampling
     // to full viewport size
     this._finalPassUpScaleToScreen = true;
+    // wrap uv stuffs
+    this._methodCWrapUV = 0;
```

Comment:
what the C stand for ? 
(not sure if color or something else?)

Reply:
typo